Sources/Concordium/Domain/Account.swift

```diff
@@ -61,7 +61,7 @@ public struct CredentialRegistrationID: Serialize, Deserialize, FromGRPC, ToGRPC
     /// Get the ``AccountAddress`` corresonding to the credential registration ID.
     public var accountAddress: AccountAddress {
         let hash = SHA256.hash(data: value)
-        return AccountAddress(Data(hash))
+        return try! AccountAddress(Data(hash))
```

Comment:
Are we completely sure that there won't be a crash here in the future?

Reply:
Yes, the `init` checks that the length of the passed data is the length of a sha256 hash 🙂